### PR TITLE
Docs: Improve VS Code readme and take it out of preview

### LINKED ISSUE #1537: [Docs] Improve VS Code extension
Add video of the extension in action, troubleshooting, etc.
This is the page that goes into the marketplace so it should be well documented.

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~~Added/Updated related documentation.~~
- ~~Added/Updated related tests.~~

## Short description of the change(s)

Fix #1537

### CODE REVIEW

**Comment by molant** [comment]
@hxlnt can you please take a look at this? 🙏

File: packages/extension-vscode/README.md [hunk 1]
```diff
@@ -7,18 +7,19 @@ based on `webhint` analysis.
...
-This extension uses your local `.hintrc` file to configure `webhint`.
-If no `.hintrc` file is found it defaults to
-[`@hint/configuration-development`][config].
+This extension will use [`@hint/configuration-development`][config] by
+default. Please take a look at that package to know what type of hints
+will be activated but to give you an idea HTML and templating systems
+(like JSX, TSX, Angular), JS, `tsconfig.json`, PWA, accessibility...
+will be enabled.
+This should be a good starting point for everyone. If you encounter
+any false possitives please [open an issue in GitHub][issue github].
```
Comment (hxlnt) [inline packages/extension-vscode/README.md]:
```suggestion
any false positives, please [open an issue in GitHub][issue github].
```

File: packages/extension-vscode/README.md [hunk 2]
```diff
@@ -7,18 +7,19 @@ based on `webhint` analysis.
...
-The extension is still in beta, please check the
-[troubleshooting section][troubleshoot] and if you cannot find an
-answer [open an issue in GitHub][issue github].
-
 ## Configuration
 
-This extension uses your local `.hintrc` file to configure `webhint`.
-If no `.hintrc` file is found it defaults to
-[`@hint/configuration-development`][config].
+This extension will use [`@hint/configuration-development`][config] by
```
Comment (hxlnt) [inline packages/extension-vscode/README.md]:
```suggestion
This extension will use the [`@hint/configuration-development`][config] configuration by
```

File: packages/extension-vscode/README.md [hunk 3]
```diff
@@ -7,18 +7,19 @@ based on `webhint` analysis.
...
+default. Please take a look at that package to know what type of hints
```
Comment (hxlnt) [inline packages/extension-vscode/README.md]:
```suggestion
default. This configuration activates hints for common features like HTML, 
```

File: packages/extension-vscode/README.md [hunk 4]
```diff
@@ -7,18 +7,19 @@ based on `webhint` analysis.
...
+will be activated but to give you an idea HTML and templating systems
```
Comment (hxlnt) [inline packages/extension-vscode/README.md]:
```suggestion
Javascript/Typescript, templating systems (JSX, Angular), and more. Refer to the package to 
```

File: packages/extension-vscode/README.md [hunk 5]
```diff
@@ -7,18 +7,19 @@ based on `webhint` analysis.
...
+(like JSX, TSX, Angular), JS, `tsconfig.json`, PWA, accessibility...
```
Comment (hxlnt) [inline packages/extension-vscode/README.md]: learn more about the default configuration.

File: packages/extension-vscode/README.md [hunk 6]
```diff
@@ -7,18 +7,19 @@ based on `webhint` analysis.
...
+will be enabled.
```
Comment (hxlnt) [inline packages/extension-vscode/README.md]:
```suggestion
```

**Pull request merged**